app/src/main/kotlin/com/wire/android/ui/home/conversations/search/SearchPeopleViewModel.kt

```diff
@@ -224,6 +252,24 @@ abstract class KnownPeopleSearchViewModel(
     abstract suspend fun searchKnownPeople(searchTerm: String): Flow<ContactSearchResult.InternalContact>
 }
 
+@OptIn(ExperimentalCoroutinesApi::class)
+abstract class SearchServicesViewModel(
```

Comment:
I would prefer to take it away to separate file for more visibility and to not make the file too big.
And can you please cover it with test